cmd/skopeo/utils.go

```diff
@@ -9,12 +9,20 @@ import (
 	"github.com/urfave/cli"
 )
 
-func contextFromGlobalOptions(c *cli.Context, credsFlag string) (*types.SystemContext, error) {
+func contextFromGlobalOptions(c *cli.Context, credsFlag, certPathFlag, tlsVerifyFlag string) (*types.SystemContext, error) {
 	ctx := &types.SystemContext{
-		RegistriesDirPath:           c.GlobalString("registries.d"),
+		RegistriesDirPath: c.GlobalString("registries.d"),
+		// DEPRECATED: keep this here for backward compatibility, but override
+		// them if per subcommand flags are provided (see below).
```

Comment:
If `--cert-path` is not interoperable with the original flag, I don’t really see the point in introducing a new version which does something else but warns.

Looking at various pastebins I’ve seen so far, `--tls-verify=false` is used pretty frequently and keeping it working would be nice.